io.openems.edge.io.shelly/bnd.bnd

```diff
@@ -10,6 +10,6 @@ Bundle-Version: 1.0.0.${tstamp}
 	io.openems.edge.common,\
 	io.openems.edge.io.api,\
 	io.openems.edge.meter.api,\
-
+	io.openems.edge.timedata.api
```

Comment:
I am changing this to `	io.openems.edge.timedata.api,\` and adding a newline. This is because prepare-commit.sh sorts the entries alphabetically. A missing trailing "\" can then possibly break the build.

Reply:
sure, understood